### ISSUE TITLE
Undefined symbols for architecture arm64

### ISSUE THREAD

**Issue by gfreezy** [val-lang/val#48]
./valc fac.val

```
Undefined symbols for architecture arm64:
  "_P3IntO5infix3u2a21_a", referenced from:
      l_F9factorial21_a in main.o
  "_P3IntO5infix3u2d21_a", referenced from:
      l_F9factorial21_a in main.o
  "_P3IntO5infix3u3c21_a", referenced from:
      l_F9factorial21_a in main.o
ld: symbol(s) not found for architecture arm64

```

**Comment by kyouko-taiga** [comment]
Thanks for reporting this bug.

It is caused by the standard library not being emitted along with the main file. This issue is currently blocked by an overhaul of the definite initialization analysis.

**Comment by spinicist** [comment]
Hello,

Any progress on this? I'd love to try out Val on my M1.

In addition, trying to compile the Hello World from the language tutorial gives the error:

`error: Hello.val:2:3: undefined name 'SourceRepresentable<Name>(value: print, range: Optional(Compiler.SourceRange(source: SourceFile(Hello.val -- file:///Users/tobias/Code/hello/, lowerBound: Swift.String.Index(_rawBits: 1442049), upperBound: Swift.String.Index(_rawBits: 1769729))))' in this scope`

This is mentioned extremely briefly at the top of #39. Should I open a separate issue?

**Comment by kyouko-taiga** [comment]
Issue has been fixed.
